src/pages/integrations/log-management/data-types/json.mdx

```diff
@@ -18,12 +18,9 @@ Send JSON logs to Logit.io via Logstash.
   curl -i -H "ApiKey: @apikey" -i -H "Content-Type: application/json" -H "LogType: default" https://@logitApiEndpoint/v2 -d '{"test":"This is a test", "Country":"United Kingdom", "Weather":"Sunny" }'
   ```
 
-  Your ApiKey can be found by doing the following:
+  When you are logged in to your Logit.io account your ApiKey should be populated automatically.
```

Comment:
This integration definitely needs some more attention but I wanted to fix the most obviously grating text as a first pass.